cr.yaml

```diff
@@ -0,0 +1,2 @@
+owner: catrielmuller
```

Comment:
Maybe the owner should be changed to freifunkMUC, since this repository is located within the freifunkMUC organization. Apart from that, everything seems to be fine so far. I would appreciate feedback from the other Helm/K8s users

Reply:
it's true, I forget push that change.